main.py

```diff
@@ -1,10 +1,64 @@
-import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
+def read_file() -> object:
+    try:
+        with open("players.json", "r") as players_file:
+            return json.load(players_file)
+    except FileNotFoundError:
+        print("Error: The file 'players.json' was not found.")
+        return None
+    except json.JSONDecodeError:
+        print("Error: Failed to parse 'players.json'."
+              " The file may be corrupted.")
```

Comment:
The print statement is split across two lines, which is unnecessary and could lead to readability issues. Consider combining these lines into one.